islandora_fits.module

```diff
@@ -93,12 +93,7 @@ function islandora_fits_ir_citationcmodel_islandora_derivative() {
  * Access callback for the FITS metadata display.
  */
 function islandora_fits_metadata_access($object) {
-  if ($object[variable_get('islandora_fits_techmd_dsid', 'TECHMD')] && user_access('view technical metadata')) {
-    return TRUE;
-  }
-  else {
-    return FALSE;
-  }
+  return islandora_datastream_access('view technical metadata', $object[variable_get('islandora_fits_techmd_dsid', 'TECHMD')]);
```

Comment:
@adam-vessey can we call islandora_datastream_access with a non existing datastream? Could $object[variable_get('islandora_fits_techmd_dsid', 'TECHMD')] be not set, which would lead
```islandora_namespace_accessible($datastream->parent->id)``` to fail on non existing property (and probably further down)

Reply:
Yes, it should be fine, as the `AbstractObject::offsetGet()` would return something that is not an object, both for:
* [`NewFedoraObject` instances](https://github.com/Islandora/tuque/blob/ceaa130ab092dbad9d3c37073cba4575687d63ea/Object.php#L624) and
* (existing) `FedoraObject` instances, via:
    * [its `::offsetGet()`](https://github.com/Islandora/tuque/blob/ceaa130ab092dbad9d3c37073cba4575687d63ea/Object.php#L895) and
    * [`::getDatastream()` methods](https://github.com/Islandora/tuque/blob/ceaa130ab092dbad9d3c37073cba4575687d63ea/Object.php#L742)

which [`islandora_datastream_access()` rejects before hitting the hooks](https://github.com/Islandora/islandora/blob/9b37dcc9b853e109ce5ba3e1ae889703803ca3a5/islandora.module#L1716-L1720).

Reply:
Right! Approved. Need to take note of which ones are ready to merge. 😄 Any preference on this? I don't feel anyone is dependant on others, so merge order could be any. Also, since it is a bug. Do you feel Ok if we go this week through the whole set, then once merged tackle the release ones? or better in pairs? @rosiel thoughts?

Reply:
Yeah, they're all independent, IIRC.

As for release PRs... seems like it might be simpler just to throw them together after the main branches gets merged, in the case there happens to be anything which comes up; however, no issue throwing them together now, if that's what's desired... will wait, though...